services/configuration/src/tests/api.rs

```diff
@@ -463,3 +513,25 @@ fn test_post_vote_tx() {
     assert_eq!(tx.hash(), info.tx_hash);
     assert!(testkit.is_tx_in_pool(&info.tx_hash));
 }
+
+#[test]
+fn test_post_vote_against_tx() {
+    let mut testkit: TestKit = TestKit::configuration_default();
+    let api = testkit.api();
+    // Commits the following configuration.
+    let new_cfg = {
+        let mut cfg = testkit.configuration_change_proposal();
+        cfg.set_actual_from(Height(10));
+        cfg.set_service_config("message", "First config change");
+        cfg.stored_configuration().clone()
+    };
+    let tx = new_tx_config_propose(&testkit.network().validators()[0], new_cfg.clone());
+    testkit.create_block_with_transactions(txvec![tx]);
```

Comment:
Nitpick: it's possible to use `create_block_with_transaction(tx)` here.

Reply:
I saw that create_block_with_transactions() is used everywhere in this file, so want to be consistent with rest of code.

Reply:
Done